components/Allowances/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { memo } from 'react';
+import i18n from 'i18n';
+import { useTranslation } from 'react-i18next';
+import { Box, Typography, useMediaQuery, useTheme } from '@mui/material';
+import { useAllowances } from 'hooks/useAllowances';
+import AllowancesMobile from 'components/Allowances/AllowancesMobile';
+import AllowancesTable from 'components/Allowances/AllowancesTable';
+import Spender from 'components/Allowances/Spender';
+import Amount from 'components/Allowances/Amount';
+import Asset from 'components/Allowances/Asset';
+
+export const ALLOWANCE_COLUMNS = [
+  {
+    title: i18n.t('Asset'),
+    sortKey: 'symbol',
+    DisplayComponent: Asset,
+  },
+  {
+    title: i18n.t('Authorized Amount'),
+    sortKey: 'allowanceUSD',
+    DisplayComponent: Amount,
+  },
+  {
+    title: i18n.t('Authorized Spender'),
+    sortKey: 'spenderName',
+    DisplayComponent: Spender,
+  },
+] as const;
```

Comment:
using the global `i18n.t` won't re-compute translations if the user changes the lang.
You can just wrap this obj in a func, or just use it inside a component and use `{ t }` from `useTranslation`.

Reply:
fixed, good catch!